app/src/utils/externalModels/evaluationModels.ts

```diff
@@ -0,0 +1,19 @@
+import { ExternalModel } from "@prisma/client";
+import { predefinedModelNames, predefinedExternalModels } from "./allModels";
+
+const evaluationModels = [
```

Comment:
Why are these different from predefined models?

Reply:
`evaluationModels` is a subset of all `predefinedExternalModels`.
For example, we do not use `gpt-3.5-turbo` for evaluations, but we will use it in other places.
Did I understand the question correctly?

Reply:
Yes, you answered correctly. However, I don't see any reason not to allow 3.5 as an evaluation model. It would be good at doing certain types of evaluations, and would cost much less. @corbt do you see any reason to omit it?

Reply:
I think we were initially discussing supporting GPT 4 only for evaluations. But I agree; I do not see any downsides to adding GPT 3.5.

Reply:
As an aside, I would either create the `evaluationModels` array inside `predefinedEvaluationModels` instead of storing it as an external variable, or change the name to be `evaluationModelNames`, since models means something else in this context.